shared/fs/banner/system-file-manager-integration-banner/container.tsx

```diff
@@ -23,15 +23,15 @@ const mapDispatchToProps = (dispatch, ownProps: OwnProps) => ({
 const ConnectedBanner = namedConnect(
   mapStateToProps,
   mapDispatchToProps,
-  (s, d, o) => ({...o, ...s, ...d}),
+  (s, d, o: OwnProps) => ({...o, ...s, ...d}),
   'SystemFileManagerIntegrationBanner'
 )(Banner)
 
 export default ConnectedBanner
 
 export const asRows = isMobile
-  ? (path: Types.Path, showBanner: boolean) => []
-  : (path: Types.Path, showBanner: boolean): Array<RowTypes.HeaderRowItem> =>
+  ? () => []
+  : (_: Types.Path, showBanner: boolean): Array<RowTypes.HeaderRowItem> =>
```

Comment:
this path parameter could just go away

Reply:
yeah some stuff i didn't plumb through all the way and cleanup all call sites, just trivial ones. hopefully the _ will stick out and people will clean it up when they're in these files